dll/win32/shell32/wine/classes.c

```diff
@@ -408,6 +408,11 @@ BOOL HCR_GetClassNameW(REFIID riid, LPWSTR szDest, DWORD len)
               if(LoadStringW(shell32_hInstance, IDS_ADMINISTRATIVETOOLS, szDest, buflen))
                   ret = TRUE;
           }
+          else if (IsEqualIID(riid, &CLSID_Internet))
```

Comment:
Shouldn't this mapping come from the registry directly? :thinking: (I have no idea about that tbh)

Reply:
According to my `MessageBox` debugging, it was needless code.

Deleted.